src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -875,15 +875,30 @@ public sealed class NgramHashingEstimator : IEstimator<NgramHashingTransformer>
         /// </summary>
         public sealed class ColumnInfo
         {
+            /// <summary>Name of the column resulting from the transformation of <see cref="InputColumnNames"/>.</summary>
```

Comment:
> /// <summary>Name of the column resulting from the transformation of <see cref="InputColumnNames"/>.</summary> [](start = 11, length = 111)

Usually the format for summary is such that <summary> open and close tags are on separate lines.
<summary>
...
</summary>

Reply:
We have started using this format when the comment can fit in one line, that way we save space. Senja has suggested this. It's usually done for things like `ColumnInfo` fields. 

---
In reply to: [258306545](https://github.com/dotnet/machinelearning/pull/2648#discussion_r258306545) [](ancestors = 258306545)